WordPress/openverse
# Use new favicon

## Fixes



Fixes #903 by @zackkrida 

## Description


This PR:
1. Replaces the .ico file with a new one I generated with `svg-to-ico` and then modified in GIMP to match the sizes of the existing favicon
2. Adds the SVG of the favicon, because I learned while doing this that many modern browsers support SVG favicons (https://caniuse.com/link-icon-svg), and in those cases the browser requests a 1.4 KB file rather than a 27.1 KB file; a small but worthwhile win :tada: 

## Testing Instructions

Run the app locally and confirm the new favicon is used. If you are on a browser that supports the svg favicon, confirm it is used instead of the ico. Then try with a browser that does not support SVG favicon (a WebKit browser https://webkit.org/downloads/).

In Firefox, you should see it request the PNG (the iPhone icon, as labelled in the code) and the SVG.

In a WebKit browser (Safari on macOS or iOS or Epiphany on Linux/Windows) you should see it only request the .ico file.

## Checklist


- [N/A] I added or updated tests for the changes I made (if applicable).
- [N/A] I added or updated documentation (if applicable).
- [N/A] I ran the DAG documentation generator (if applicable).

[best_practices]:
  https://git-scm.com/book/en/v2/Distributed-Git-Contributing-to-a-Project#_commit_guidelines

## Developer Certificate of Origin


<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>
